.typos.toml

```diff
@@ -23,6 +23,8 @@ encrypter = "encrypter"       # Used by the `ring` crate
 nin = "nin"                   # National identification number, a field used by PayU connector
 substituters = "substituters" # Present in `flake.nix`
 unsuccess = "unsuccess"       # Used in cryptopay request
+ba = "ba" # ignore minor commit conversions
```

Comment:
@SanchithHegde  - are these needed ?

Reply:
This is only to pass the spell checks that run on CI. We either have to do something like this, not include commit hashes in the changelog file, or exclude the changelog file from being spell-checked at all. I chose to go with the first option.